treerec/tskit/tables.c

```diff
@@ -3523,7 +3516,10 @@ table_sorter_sort_edges(table_sorter_t *self, tsk_size_t start)
         e->child = self->edges->child[k];
         e->time = self->nodes->time[e->parent];
     }
-    qsort(sorted_edges, n, sizeof(edge_sort_t), cmp_edge);
+    
+    //Commented out quick sort(serial) for now. Will add switch for serial execution once Ben verifies this part 
```

Comment:
Might as well add this switch; I'm not sure all the details are sorted out yet, but let's try to get the code into as final a state as we can.  No harm in adding the #ifdef here and calling qsort when we're single-threaded (if that's what you plan to do); it will make the overall design plan clearer.